ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -206,24 +206,21 @@ class H2OGridSearch(override val uid: String) extends Estimator[H2OMOJOModel]
       (m, extractMetrics(m).find(_._1 == metric).get._2)
     }
 
-    val ordering = if (getSelectBestModelBy() == H2OGridSearchMetric.AUTO.name()) {
-      logWarning("'selectBestModelBy' parameter is specified to 'AUTO', but you also specified 'selectBestModelDecreasing'." +
```

Comment:
Make it more transparent by making creating Enum for ordering which says auto as well.

By default both values are auto and we can avoid this warninig which was printed all the time